pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go

```diff
@@ -0,0 +1,145 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver && orchestrator
+// +build kubeapiserver,orchestrator
+
+package orchestrator
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/collectors"
+	"github.com/DataDog/datadog-agent/pkg/orchestrator"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"k8s.io/client-go/tools/cache"
```

Comment:
do you mean splitting dd imports and non-std imports?

Reply:
Yes, but tbh I am not 100% sure whether we follow this pattern here in datadog-agent as well.

Reply:
Right, I see how it could make sense as a coding style. I have two problems though:

On the tooling side: trying to setup goimports to use `-local=X` which is supposed to do this automatically doesn't work on my side 😢 

On the adoption: I'm seeing either the
```
std

non-std
```
which is go default

and

```
std

non-std (internal, i.e. company repos)

non-std (external)
```

Patterns in most places so if someone doesn't have the right tooling this will be adding imports to the section dedicated to company imports and might be a pain during reviews.

Reply:
yeah indeed, no real big opinion from my side here. AFAIK at most places we try the second one but it is not 100% adhered. 

`goimports` works fine for me with `-w <FILEPATH>` 😆